package.json

```diff
@@ -5,6 +5,7 @@
   "version": "1.0.25",
   "scripts": {
     "test": "grunt test",
+    "test:karma": "karma start tests/karma/single.conf.js",
```

Comment:
use local karma package instead of global package
./node_modules/karma/bin/karma